<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointCellDataSelector</class>
 <widget class="QWidget" name="PointCellDataSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>103</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>Point Data</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="ProductTreeWidget" name="PointArrayStatus" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="tabKeyNavigation" stdset="0">
          <bool>true</bool>
         </property>
         <property name="alternatingRowColors" stdset="0">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="iconSize" stdset="0">
          <size>
           <width>5</width>
           <height>5</height>
          </size>
         </property>
         <property name="rootIsDecorated" stdset="0">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable" stdset="0">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled" stdset="0">
          <bool>false</bool>
         </property>
         <property name="animated" stdset="0">
          <bool>false</bool>
         </property>
         <property name="headerHidden" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Cell Data</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="ProductTreeWidget" name="CellArrayStatus" native="true">
         <property name="tabKeyNavigation" stdset="0">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors" stdset="0">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated" stdset="0">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable" stdset="0">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled" stdset="0">
          <bool>false</bool>
         </property>
         <property name="animated" stdset="0">
          <bool>false</bool>
         </property>
         <property name="headerHidden" stdset="0">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProductTreeWidget</class>
   <extends>QWidget</extends>
   <header>QAllTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
